react_ujs/index.js

```diff
@@ -6,6 +6,8 @@ var detectEvents = require("./src/events/detect")
 var constructorFromGlobal = require("./src/getConstructor/fromGlobal")
 var constructorFromRequireContextWithGlobalFallback = require("./src/getConstructor/fromRequireContextWithGlobalFallback")
 
+var renderWithReactDOM = require("./src/renderComponent/withReactDOM")
```

Comment:
folder structure and import pattern is taking reference from `getConstructor` for consistency